integration-tests/http/__tests__/product/admin/product.spec.ts

```diff
@@ -2356,6 +2356,125 @@ medusaIntegrationTestRunner({
           )
         })
 
+        it("updates variant to manage_inventory false and unlinks inventory items", async () => {
```

Comment:
suggestion: If we could always start test case with `should` just for consistency :)